src/jsifier.mjs

```diff
@@ -45,6 +45,10 @@ const addedLibraryItems = {};
 
 const extraLibraryFuncs = [];
 
+// Experimental feature to check for invalid __deps entries. Modify this line
+// locally to try it out.
+const CHECK_DEPS = true;
```

Comment:
I plan on setting this to false before landing it.

We can look into enabling more widely as a followup.